packages/desktop/components/popups/claimDelegationRewardsPopup.svelte

```diff
@@ -0,0 +1,90 @@
+<script lang="ts">
+    import { Button, Text, FontWeight, TextType, KeyValueBox } from '@ui'
+    import { localize } from '@core/i18n'
+    import { closePopup, updatePopupProps } from '@auxiliary/popup'
+    import { getDefaultTransactionOptions, selectedWallet, selectedWalletId } from '@core/wallet'
+    import { checkActiveProfileAuth } from '@core/profile/actions'
+    import { ManaBox } from '@components'
+    import { onMount } from 'svelte'
+    import { handleError } from '@core/error/handlers'
+    import { updateActiveWallet } from '@core/profile'
+    import { ITransactionInfoToCalculateManaCost } from '@core/network'
+
+    export let _onMount: (..._: any[]) => Promise<void> = async () => {}
+    export let delegationId: string
+    export let rewards: number
+    export let isBusy = false
+
+    let hasEnoughMana = false
+    const transactionInfo: ITransactionInfoToCalculateManaCost = {}
+
+    async function onConfirmClick(): Promise<void> {
+        isBusy = true
+        try {
+            updatePopupProps({ isBusy })
+            await checkActiveProfileAuth(burnDelegationOutput, { stronghold: true })
+        } catch (error) {
+            console.error(error)
+        } finally {
+            isBusy = false
+        }
+    }
+
+    async function burnDelegationOutput(): Promise<void> {
+        try {
+            await $selectedWallet.burn({ delegations: [delegationId] }, getDefaultTransactionOptions())
+            updateActiveWallet($selectedWalletId, {
+                hasDelegationTransactionInProgress: true,
+                isTransferring: true,
+            })
```

Comment:
```suggestion
            closePopup()
```

`hasDelegationTransactionInProgress` is used to create a new delegation output
If you want, you can create a new filed in the store and close the popup in `handleSpentOutputEvent`, but imo it is not necessary and it can be closed here